cypress/integration/percy/article.amp.spec.js

```diff
@@ -13,6 +13,8 @@ describe('For AMP', function() {
         const { url, pillar, designType } = article;
         it(`It should load ${designType} articles under the ${pillar} pillar`, function() {
             cy.visit(`AMPArticle?url=${url}`, visitOptions);
+            // Don't include the privacy banner in snapshots
+            cy.contains('OK with that').click();
```

Comment:
It will be interesting to see how brittle these tests are. But it seems like low-effort to modify them in any case.

Reply:
True. The [docs](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) are pretty good on this subject but it's obviously subjective. This text felt like something that was reasonably static but we could also use a `data-*` selector later if we wanted more confidence.